src/libsystemd/sd-journal/journal-verify.c

```diff
@@ -824,6 +824,35 @@ static int verify_entry_array(
         return 0;
 }
 
+static int verify_hash_table(
+                Object *o, uint64_t p, uint64_t *n_hash_tables, uint64_t header_offset, uint64_t header_size) {
+
+        if (*n_hash_tables > 1) {
+                error(p, "More than one data hash table: %" PRIu64, *n_hash_tables);
```

Comment:
It's not really a data hash table if you verify the field hash table... 🤔

Reply:
Yikes, fixed